new-site/src/docs/foundation/guidelines/checkbox-radiobutton-toggle.mdx

```diff
@@ -1,11 +1,12 @@
 ---
-slug: "/guidelines/checkbox-radiobutton-toggle"
+slug: "/foundation/guidelines/checkbox-radiobutton-toggle"
 title: "Checkboxes, radio buttons, or toggles?"
+nav_title: 'Checkboxes, radio buttons, or toggles?'
 ---
 
 import { Notification, Checkbox, RadioButton, ToggleButton, SelectionGroup, Link } from "hds-react";
-import LeadParagraph from '../../components/LeadParagraph';
-import PlaygroundPreview from '../../components/Playground';
+import LeadParagraph from '../../../components/LeadParagraph';
+import PlaygroundPreview from '../../../components/Playground';
 
 # Checkboxes, radio buttons, or toggles?
 
```

Comment:
Component links of this page are still pointing into elements folder

Reply:
- [ x ] Fixed broken links in the Checkbox/RadioButton/Toggle guideline page